src/main.rs

```diff
@@ -816,6 +832,12 @@ compatibility with the original codemod.",
                 .long("hidden")
                 .help("Search hidden files.")
         )
+        .arg(
+            Arg::with_name("no_ignore")
+                .short("u")
```

Comment:
mildly concerned that we're signing up for a bug report that `-u -u` doesn't do the same thing as it does in ripgrep, but I think I'm OK with that for now.

Reply:
I'm open to change this to match ripgrep's [behaviour](https://github.com/BurntSushi/ripgrep/blob/94305125ef33b86151b6cd2ce2b33d641f6b6ac3/GUIDE.md?plain=1#L204), (expect the part about binary files of course).

> That is, -u will disable .gitignore handling, -uu will search hidden files

Reply:
Although, it might be worth mentioning my tiny annoyance with the `-u` flag in `ripgrep` vs `fd-find`. Most times I forget that `fd -u` is NOT the same as `rg -u` but `rg -u`.

> To really search all files and directories, simply combine the hidden and ignore features to show everything (-HI) or use -u/--unrestricted.

https://github.com/sharkdp/fd?tab=readme-ov-file#hidden-and-ignored-files

Reply:
Should I just remove the short option for now? The long option should be unambiguous.